pkg/kubelet/watchdog/watchdog_linux.go

```diff
@@ -104,25 +104,29 @@ func NewHealthChecker(syncLoop syncLoopHealthChecker, opts ...Option) (HealthChe
 		return nil, fmt.Errorf("configure watchdog timeout too small: %v", watchdogVal)
 	}
 
-	// The health checks performed by checkers are the same as those for "/healthz".
-	checkers := []healthz.HealthChecker{
-		healthz.PingHealthz,
-		healthz.LogHealthz,
-		healthz.NamedCheck("syncloop", syncLoop.SyncLoopHealthCheck),
-	}
 	retryBackoff := wait.Backoff{
 		Duration: time.Second,
 		Factor:   2.0,
 		Jitter:   0.1,
 		Steps:    2,
 	}
-	hc.checkers = append(hc.checkers, checkers...)
 	hc.retryBackoff = retryBackoff
 	hc.interval = watchdogVal / 2
 
 	return hc, nil
 }
 
+func (hc *healthChecker) AddHealthCheckers(syncLoop syncLoopHealthChecker, checkers []healthz.HealthChecker) {
```

Comment:
the semantic of this is not clear. Typically `Add` will be ok to use multiple times. This one is more like `Initialize` or `Enable`

Reply:
Fixed